client/stylesheets/abstracts/_variables.scss

```diff
@@ -33,3 +33,7 @@ $alert-red: $error-red;
 // WordPress defaults
 $adminbar-height: 32px;
 $adminbar-height-mobile: 46px;
+
+// Muriel
+$muriel-card-box-shadow: 0 1px 3px rgba(0, 0, 0, 0.2), 0 2px 2px rgba(0, 0, 0, 0.12),
```

Comment:
This shadow doesn't match the 1dp elevation styles.

It would be cool to establish vars for all different elevation heights.

Reply:
@joshuatf @jameskoster - should I be concerned with the type styles and padding for card headers at this point? Or are we just focused on the actual card styles?

My concern with the size of the card header title in this example is we'll run out of space for things like charts and tabular data where we have other inputs and controls included in the card header. Here's an example: 

<img width="1354" alt="Screen Shot 2019-05-17 at 3 34 54 PM" src="https://user-images.githubusercontent.com/4500952/57951956-b572e700-78b9-11e9-9e78-bb8c0b30b311.png">

if we use the new Muriel type scale, I think we'd want to use either "Subtitle"  or "Subtitle Small"

<img width="1422" alt="Screen Shot 2019-05-17 at 3 39 00 PM" src="https://user-images.githubusercontent.com/4500952/57952038-f79c2880-78b9-11e9-8c08-56bace6c195e.png">

This is an exploration I was working for advanced filters that uses "Subtitle" in the card header. The overall card header height is 72px.

<img width="1130" alt="Screen Shot 2019-05-17 at 3 40 56 PM" src="https://user-images.githubusercontent.com/4500952/57952131-434ed200-78ba-11e9-977d-81d9ce62c0c5.png">

Reply:
@LevinMedia and I spoke about discrepancies in Card styles between Muriel and wc-admin.  Opened an issue [here](https://github.com/woocommerce/woocommerce-admin/issues/2259) to discuss further.

Reply:
@jameskoster Messaged you in Slack about this as well, but the Figma designs for the Card component and the onboarding cards look to have this box-shadow and not the 1dp shadow.

Are those styles wrong or outdated?  Happy to tackle this in a follow-up as well if we don't have the answer now so we can unblock other issues that need this component.

Reply:
I replied on Slack, but for anyone else following along here - I've updated the shadows in Figma to match the Material elevation styles as closely as possible. However, Figma doesn't account for the `spread` property of CSS drop-shadows, so I suggest using https://material-components.github.io/material-components-web-catalog/#/component/elevation for the actual code.

Reply:
@jameskoster Thanks for the update.  Updated that box shadow here.  Could you review when you get a chance?